motif-docs/pages/guides/advanced/swr.mdx

```diff
@@ -10,7 +10,7 @@ Flightcontrol has an option to add `stale-while-revalidate` cache-control header
 
 The [`stale-while-revalidate cache-control header`](https://datatracker.ietf.org/doc/html/rfc5861#section-3) is useful when you care more about the speed of all user’s requests more than you care about users seeing the latest data. When used, the CDN will will serve stale content to the user while refetching the latest data in the background so that the next user will get the latest data.
 
-## When to use?
+## [When to use?](#)
```

Comment:
Ok, let's revert these manual additions, and take up auto links with another task